src/Widgets/WindowSwitcher/WindowSwitcher.vala

```diff
@@ -156,20 +162,38 @@ public class Gala.WindowSwitcher : CanvasActor {
         ctx.clip ();
         ctx.reset_clip ();
 
-        var background_color = Drawing.Color.LIGHT_BACKGROUND;
-        if (style_manager.prefers_color_scheme == Drawing.StyleManager.ColorScheme.DARK) {
-            background_color = Drawing.Color.DARK_BACKGROUND;
-        }
-
         ctx.set_operator (Cairo.Operator.SOURCE);
+
+        Drawing.Utilities.cairo_rounded_rectangle (ctx, 0.5, 0.5, width - 1, height - 1, InternalUtils.scale_to_int (9, scaling_factor));
```

Comment:
I'm not sure whether the 0.5, - 1 here or -3 further below should be scaled as well but I guess with numbers this small it's not so important anyways?

Reply:
Ah yeah you’re right this should probably be scaled so we don’t get tiny borders on hidpi. Will fix!